chatsky/pipeline/pipeline/pipeline.py

```diff
@@ -294,11 +295,27 @@ def to_tuple(i):
             if isinstance(i, list):
                 return tuple(i)
             return i
+
+        def dict2slots(slots_dict: dict):
+            slots = defaultdict(dict)
+            for group_slot_name, group_slot_value in slots_dict.items():
+                for slot_type, slot_value in group_slot_value.items():
+                    if slot_class := globals().get(slot_type):
+                        slots[group_slot_name] = GroupSlot(**{k: slot_class(**v) for k, v in slot_value.items()})
+                    else:
+                        nested_group_slots = dict2slots({slot_type: slot_value})
+                        slots[group_slot_name] = GroupSlot(**nested_group_slots)
+            return slots
```

Comment:
I think we can remove this function.
If you call `JSONImporter.replace_script_objects` on `CONFIG` and write slots in this way:
```yaml
  slots:
    person:
      username:
        chatsky.slots.RegexpSlot:
          regexp: r"username is ([a-zA-Z]+)"
          match_group_idx: 1
      email:
        chatsky.slots.RegexpSlot:
          regexp: r"email is ([a-z@\.A-Z]+)"
          match_group_idx: 1
```
then `GroupSlot.model_validate` call inside `Pipeline` should automatically parse all group slots and `replace_script_objects` would prepare all value slots.

This approach is more consistent with the way script is written.